README.md

```diff
@@ -50,3 +50,6 @@ where <kbd>path/to/record</kbd> is the file path to the record you wish to load.
 ``` sh
 $ python load_record.py /data/eads/lae/LAE001.EAD.xml
 ```
+## Logs
+For debugging eXist XSLT errors: `/usr/local/lib/exist/webapp/WEB-INF/logs`
```

Comment:
Could you format these so that they appear on two lines?

Reply:
👍 Done!